lib/sidekiq_unique_jobs/middleware/client/strategies/unique.rb

```diff
@@ -75,7 +75,7 @@ def old_unique_for?
 
           def new_unique_for?
             connection do |conn|
-              return conn.set(payload_hash, item['at'] ? 2 : 1, nx: true, ex: expires_at)
+              return conn.set(payload_hash, item['jid'], nx: true, ex: expires_at)
```

Comment:
How is jid used? The original functionality ensured that we can use both scheduled (timed) jobs and the regular immediate queue. Won't this be a problem now in regards to uniqueness? I thought the 2 and 1 is used by sidekiq to determine if it is a scheduled type of job or a regular? Am I missing something @pik ?